src/common/Version.cpp

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2019 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "oboe/Version.h"
+
+namespace oboe {
+	char kVersionText[] = "OboeVersion" OBOE_VERSION_TEXT;
```

Comment:
Someone might come along later and optimize this.
Maybe add a comment that says it is coded this way so that the version string can be read from the app binary.